android/app/src/main/java/com/microsoft/codepush/common/CodePushStatusReportIdentifier.java

```diff
@@ -0,0 +1,116 @@
+package com.microsoft.codepush.common;
+
+import android.support.annotation.NonNull;
+
+/**
+ * Identifier for status report saved on users device.
+ * Basically it is used in {@link com.microsoft.codepush.react.managers.CodePushTelemetryManager}
+ * for identifying status reports saved on users device. Identifier consist of two parts: deployment key and version label.
+ * If both are exist it means that update report was saved, if only version label exists it means that binary report was saved.
+ * There shouldn't be a situation when version label doesn't exist.
+ * <p>
+ * Identifier can be serialized into string for saving on device and deserialized from it.
+ */
+public class CodePushStatusReportIdentifier {
+
+    /**
+     * Deployment key.
+     */
+    private String deploymentKey;
+
+    /**
+     * Version label.
+     */
+    private String versionLabel;
+
+    /**
+     * Constructor for identifier of binary report.
+     *
+     * @param versionLabel version label.
+     */
+    public CodePushStatusReportIdentifier(@NonNull String versionLabel) {
+        this.versionLabel = versionLabel;
+    }
+
+    /**
+     * Constructor for identifier of update report.
```

Comment:
Please use default naming structure here: "Creates an instance of {@link ...} ..."

Reply:
@annakocheshkova Got it, I'll fix it.